src/_includes/components/footer.njk

```diff
@@ -45,10 +48,25 @@
             },
             body
           }).then((res) => {
-            form.reset();
-            success.classList.remove("hidden");
+            if (res.status === 400) throw Error("The email you provided was invalid");
```

Comment:
Since you're doing `return respondWith(callback,400,"This is an invalid email")` in the function, can you use that response instead of repeating the "The email you provided was invalid" message here? That way, if we return a different error message from the function we won't have to update the client code.